workflow/generate_workflows.sh

```diff
@@ -0,0 +1,532 @@
+#!/usr/bin/env bash
+
+###
+function _usage() {
+   cat <<-EOF
```

Comment:
`<<-` won't work as expected without using tabs for indenting
```suggestion
   cat << EOF
```